templates/index.html.hbs

```diff
@@ -35,5 +35,8 @@
     </div>
   </div>
 </form>
+<script>
+	document.getElementsByName('f')[0].email.select();
```

Comment:
The `email` field has a name, you should be able to use that without giving the form a name.

However, this actually does not need any JavaScript: Please use the [`autofocus` attribute](https://www.w3schools.com/tags/att_input_autofocus.asp).